osu.Game/Screens/Play/BreakOverlay.cs

```diff
@@ -125,6 +127,7 @@ private void initializeBreaks()
 
                 using (BeginAbsoluteSequence(b.StartTime, true))
                 {
+                    Schedule(() => IsActive = true);
```

Comment:
This won't work with rewinding, which I believe we'll want during gameplay at some point. Maybe iterate over all the break periods instead?